.github/workflows/fast-forward-branch.yml

```diff
@@ -4,11 +4,9 @@ on:
     inputs:
       ref:
         description: 'The branch name or commit to fast-forward from'
-        default: 'master'
         type: string
       to_branch:
         description: 'The branch name to fast-forward to'
-        default: 'managed-services'
```

Comment:
why remove the defaults?

Reply:
I just changed my mind and thought it was better to remove the defaults so it fails if the inputs are not being sent correctly, which is what was happening.